File: ObjemHadice.py
import os
while True:
        print("Priemery hadic:\n1'' = 3,2cm\n1/2'' = 1,6cm\n3/4'' = 2,5cm\n5/4'' = 4,0cm")
        r = float(input("Zadaj priemer hadice v cm: "))
        h = float(input("Zadaj dlzku hadice v cm: "))
        volume = ((22 * (r / 2) * (r / 2) * h) / 7)
        liter = volume / 1000

        print("\nObjem hadice je: %.2f"  % (volume), " cm'3\n", "Do hadice sa zmesti: %.2f" % (liter)," L vody.")
        next = input('Chces skoncit? a/n: \n')
        if next == 'a':
                break
os.system("pause")

File: ObjemHadice.java
import java.util.Scanner;

public class ObjemHadice {

    public static void main(String[] args) throws Exception {

        Scanner s = new Scanner(System.in);

        System.out.println("Priemery hadic:\n1'' = 3,2cm\n1/2'' = 1,6cm\n3/4'' = 2,5cm\n5/4'' = 4,0cm");
        System.out.println("Zadaj priemer hadice v cm:");
        double r = s.nextDouble();
        System.out.println("Zadaj dlzku hadice v cm:");
        double h = s.nextDouble();

        double volume = ((22 * (r / 2) * (r / 2) * h) / 7);
        double liter = volume / 1000;

        double volume2 = (double) (Math.round(volume * 100.0) / 100.0);
        double liter2 = (double) (Math.round(liter * 100.0) / 100.0);

        System.out.println("Objem hadice je: " + volume2 + " cm'3\n" + "Do hadice sa zmesti: " + liter2 + " L vody.");

        s.close();
    }

}